include/dlaf/multiplication/hermitian.h

```diff
@@ -31,10 +73,10 @@ namespace dlaf {
 /// @param mat_a contains the hermitian matrix A. Only the tiles of the matrix which contain the upper or
 /// the lower triangular part which represent the Hermitian matrix (depending on the value of uplo)
 /// are accessed in read-only mode (the elements are not modified),
-/// @pre @p mat_b is not distributed
-/// @pre @p mat_b has size (N x M)
-/// @pre @p mat_b has blocksize (NB x NB)
-/// @pre @p mat_b has tilesize (NB x NB)
+/// @pre @p mat_a is not distributed
+/// @pre @p mat_a has size (N x M)
+/// @pre @p mat_a has blocksize (NB x NB)
+/// @pre @p mat_a has tilesize (NB x NB)
```

Comment:
Nice catch!

Reply:
I think it's fine to add them here, but if you prefer to have this in a separate PR I can do that too, I don't mind.

Reply:
I'm fine with leaving them here.